.env

```diff
@@ -0,0 +1 @@
+PORT=7080
```

Comment:
Is the PORT variable needed on cosmos? Asking as it's not used elsewhere within this repo

Reply:
It's used for the PORT (`process.env.PORT`) here: https://github.com/bbc/simorgh/pull/20/files#diff-1fdf421c05c1140f6d71444ea2b27638R5

Reply:
Gotcha 👌